StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -123,6 +120,22 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
             }
         }
 
+        // TESTING, TOBEREMOVED after review
+        static int aaa = 0;
+        if (window->WasSpacePressed())
+        {
+            if (aaa++ % 2 == 0)
+            {
+               
+            }
+            else
+            {
+                
+            }
+            
+            sceneManager->SetScenes(1, activeScenes);
+        }
```

Comment:
I modified this test to swap between my (jocke) and leos testScenes. When I spammed space, the program hung. I expect an issue to be WaitForGPU when you setScene.